gui/WorldView.cxx

```diff
@@ -85,10 +87,47 @@
 #include <QToolButton>
 #include <QWidgetAction>
 
+#include <array>
+
 using namespace LandmarkArrays;
 
 QTE_IMPLEMENT_D_FUNC(WorldView)
 
+
+namespace {
+vtkSmartPointer<vtkColorTransferFunction> setupBluesColorPalette(double* range)
+{
+  auto lut = vtkSmartPointer<vtkColorTransferFunction>::New();
+  std::array<std::array<double, 3>, 17> blues = {{
+      {0.031373, 0.188235, 0.419608},
+      {0.031373, 0.253195, 0.516063},
+      {0.031757, 0.318139, 0.612149},
+      {0.080969, 0.38113, 0.661361},
+      {0.130427, 0.444152, 0.710327},
+      {0.195386, 0.509112, 0.743791},
+      {0.260715, 0.573841, 0.777209},
+      {0.341423, 0.628958, 0.808704},
+      {0.422745, 0.684075, 0.839892},
+      {0.523137, 0.739193, 0.861546},
+      {0.622684, 0.793464, 0.883429},
+      {0.701423, 0.826928, 0.910988},
+      {0.778685, 0.8603, 0.937993},
+      {0.825928, 0.891795, 0.953741},
+      {0.87328, 0.923291, 0.969489},
+      {0.922491, 0.954787, 0.985236},
+      {0.968627, 0.984314, 1}}};
+  for (size_t i = 0; i < blues.size(); ++i)
```

Comment:
```suggestion
  for (auto const i : qtIndexRange(blues.size()))
```

(Note: `qtIndexRange` and `kwiver::vital::range::iota` are the same thing, but TS currently has several uses of the former and none of the latter, and `qtIndexRange.h` is already included.)

Reply:
This does not bring in much: it's longer and uses qtIndexRange which I never seen so its not necessarily clearer. So I would not do this.

Reply:
It makes the value immutable. It makes it so you don't need to know the correct type (which, yes, *is* an issue because there are classes floating around that use `int` for indexing.)

Would you, in Python, write a `for` loop like this? Or would you write `for i in range(blues.size())`? Would you decline to use that because "I've never seen [range]"?

Reply:
Oh, I see. It makes it Python like. Didn't realize this initially. Still don't see why would you do this when there is a standard way to do this in C/C++.

Reply:
The "standard way to do this in C/C++" (well, in C++ anyway) is [`std::views::iota`](https://en.cppreference.com/w/cpp/ranges/iota_view), which a) unfortunately isn't available yet¹ and b) unfortunately requires specifying the starting index. And possibly also requires specifying the type explicitly. (I'm not entirely clear why a more direct equivalent hasn't been standardized, given how frequently it's needed.)

(¹ Requires C++20, which we are some time from being able to use.)

Reply:
The humble for loop to the rescue 😄